spec/factories/refund_responses.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  sequence :adyen_refunds_response do
+    OpenStruct.new(
+      response: {
+        "merchantAccount" => "#{SecureRandom.uuid}",
+        "pspReference" => "#{SecureRandom.uuid}",
+        "paymentPspReference" => "#{SecureRandom.uuid}",
+        "status" => "received",
+        "amount" => {
+            "currency" => "CHF",
+            "value" => 134
+        }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.